migrations/0001_schema.sql

```diff
@@ -95,4 +95,11 @@ CREATE TABLE embedding(
 
     PRIMARY KEY (id),
     FOREIGN KEY(chunk_id) REFERENCES chunk (id)
-);
\ No newline at end of file
+);
+
+-- Default collection
+INSERT INTO collection (name, text_embedding_model, text_distance_metric) VALUES ('main', 'openai:text-embedding-ada-002', 'cosine');
+CREATE INDEX embedding_collection_1_index
+ON embedding
+USING hnsw ((embedding::vector(1536)) vector_cosine_ops)
+WHERE collection_id = 1;
```

Comment:
Creating a default "main" collection. This makes it easier to use the API in many cases when you don't care about segregating documents from each other.